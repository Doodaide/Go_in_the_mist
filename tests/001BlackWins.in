place A1
place A2
place B1
place B2
place C1
place C2
place D1
place D2
place E1