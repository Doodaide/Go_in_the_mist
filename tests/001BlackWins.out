Black wins!
A1A2B1B2C1C2D1D2E1
Thank you for playing!
